### PR TITLE
Feature/get item endpoint poc

### PR DESCRIPTION
**Pull request by rsb007**
Test coverage  is increased  and code refactored

### CODE REVIEW

File: actix_poc/src/env_set_up/connection.rs [hunk 1]
```diff
@@ -1,14 +1,17 @@
 use cdrs::authenticators::NoneAuthenticator;
-use cdrs::cluster::{ClusterTcpConfig, NodeTcpConfigBuilder, TcpConnectionPool};
 use cdrs::cluster::session::{new as new_session, Session};
+use cdrs::cluster::NodeTcpConfig;
+use cdrs::cluster::{ClusterTcpConfig, NodeTcpConfigBuilder, TcpConnectionPool};
 use cdrs::load_balancing::RoundRobin;
+use constants::constant::DATABASE_PORT_ADDRESS;
```
Comment (amitayadav) [inline actix_poc/src/env_set_up/connection.rs]: Use appropriate indentation.

File: actix_poc/src/controller/handler.rs [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+use models::model::Student;
+use repository::{
+    delete::delete_student, display::select_student, insert::insert_student, update::update_student,
+};
+
+pub struct AppState {
+    pub session: CurrentSession,
+}
+
+pub fn insert(data: State<AppState>, student: Json<Student>) -> Result<&'static str, CustomError> {
```
Comment (amitayadav) [inline actix_poc/src/controller/handler.rs]: Add docs across all modules.

File: actix_poc/src/main.rs [hunk 3]
```diff
@@ -1,40 +1,46 @@
...
+                r.method(http::Method::GET).with(show)
+            })
+            .resource("/delete/{roll_no}", |r| {
+                r.method(http::Method::DELETE).with(delete)
+            })
+            .resource("/update/{roll_no}", |r| {
+                r.method(http::Method::PUT).with(update)
+            })
     });
     server = if let Some(l) = listenfd.take_tcp_listener(0).unwrap() {
```
Comment (amitayadav) [inline actix_poc/src/main.rs]: Use meaningful variable's name.

File: rust_user_service/src/main.rs [hunk 4]
```diff
@@ -17,6 +17,7 @@ use user_service::user_service_impl::constants::constant::TAKE_FIRST;
 use user_service::user_service_impl::constants::constant::DEBUG_LEVEL_KEY;
 use user_service::user_service_impl::constants::constant::DEBUG_LEVEL_VALUE;
 
+#[cfg_attr(tarpaulin, skip)]
 fn main() {
     let _user_store: MemoryEventStore = MemoryEventStore::new();
```
Comment (amitayadav) [inline rust_user_service/src/main.rs]: No need of memory store.

File: rust_user_service/src/user_service_impl/controller/error.rs [hunk 5]
```diff
@@ -11,6 +11,7 @@ pub enum CustomError {
 }
 
 impl error::ResponseError for CustomError {
+    #[cfg_attr(tarpaulin, skip)]
```
Comment (amitayadav) [inline rust_user_service/src/user_service_impl/controller/error.rs]: cover the code instead of skipping

**Comment by rsb007** [comment]
code is megre in develop

**Pull request closed without merging**